compiler/rustc_expand/src/expand.rs

```diff
@@ -2289,15 +2332,12 @@ impl<'a, 'b> MutVisitor for InvocationCollector<'a, 'b> {
         if let Some(attr) = node.attrs.first() {
             self.cfg().maybe_emit_expr_attr_err(attr);
         }
-        self.visit_node(node)
+        self.visit_node(node);
```

Comment:
```suggestion
        self.visit_node(node)
```
Nit: could you avoid unnecessary changes?